wolfcrypt/src/logging.c

```diff
@@ -471,26 +471,48 @@ void WOLFSSL_BUFFER(const byte* buffer, word32 length)
 
     while (buflen > 0) {
         int bufidx = 0;
-        XSNPRINTF(&line[bufidx], sizeof(line)-bufidx, "\t");
+        if (XSNPRINTF(&line[bufidx], sizeof(line)-bufidx, "\t")
+            >= (int)sizeof(line) - bufidx)
+        {
+            return;
```

Comment:
We should log something so that we can see that something should have gone out.

Reply:
OK OK, I was just being lazy :stuck_out_tongue:    fixedit.